bundles/org.openhab.binding.draytonwiser/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,411 @@
+		<item-type>Number</item-type>
+		<label>Signal RSSI</label>
+		<description>The reported network signal RSSI</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="signalLQI-channel" advanced="true">
+		<item-type>Number</item-type>
+		<label>Signal LQI</label>
+		<description>The reported network signal LQI</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="wiserSignalStrength-channel" advanced="true">
+		<item-type>String</item-type>
+		<label>Signal Strength (Wiser)</label>
+		<description>The reported network signal strength</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="wiserBatteryLevel-channel" advanced="true">
+		<item-type>String</item-type>
+		<label>Battery Level (Wiser)</label>
+		<description>Current Battery Level for the device</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="heatChannelDemand-channel">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Heat Demand</label>
+		<description>Current heat demand in %</description>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="demandState-channel">
+		<item-type>Switch</item-type>
+		<label>Requesting Heat</label>
+		<description>Current channel is requesting heat</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="awayModeState-channel">
+		<item-type>Switch</item-type>
+		<label>Away Mode Active</label>
+		<description>Away mode has been activated</description>
+	</channel-type>
+
+	<channel-type id="awayModeSetPoint-channel" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>Away Mode Set Point</label>
+		<description>Away Mode Set Point</description>
+		<category>Temperature</category>
+		<state readOnly="false" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="ecoModeState-channel">
+		<item-type>Switch</item-type>
+		<label>Eco Mode Enabled</label>
+		<description>Eco Mode activation state (EcoIQ)</description>
+	</channel-type>
+
+	<channel-type id="manualModeState-channel">
+		<item-type>Switch</item-type>
+		<label>Manual Mode Enabled</label>
+		<description>Manual Mode activation state</description>
+	</channel-type>
+
+	<channel-type id="zigbeeConnected-channel" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Device Connected</label>
+		<description>Is the device still connected to the ZigBee network</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="hotWaterSetPoint-channel">
+		<item-type>Switch</item-type>
+		<label>Set Point</label>
+		<description>Hot water heating set point switch</description>
+	</channel-type>
+
+	<channel-type id="boostDuration-channel">
+		<item-type>Number</item-type>
+		<label>Boost Duration</label>
+		<description>Duration to boost hot water for</description>
+		<state pattern="%.1f h" min="0" max="3" step="0.5"/>
```

Comment:
I would make the same suggestion here, but since the core doesn't properly support handling QuantityType input ranges I think you should leave this as is for now. At the very least though you should specify the expected input unit in the description (in this case hours).

Reply:
Fixed in 8aa384f